frontend/test/metabase/scenarios/visualizations/reproductions/14148-pivot-table-postgres.cy.spec.js

```diff
@@ -10,35 +10,37 @@ describe("issue 14148", () => {
   beforeEach(() => {
     restore("postgres-12");
     cy.signInAsAdmin();
-
-    cy.intercept("POST", "/api/dataset/pivot").as("pivotDataset");
   });
 
   it("postgres should display pivot tables (metabase#14148)", () => {
     withDatabase(PG_DB_ID, ({ PEOPLE, PEOPLE_ID }) =>
-      visitQuestionAdhoc({
-        display: "pivot",
-        dataset_query: {
-          type: "query",
-          database: PG_DB_ID,
-          query: {
-            "source-table": PEOPLE_ID,
-            aggregation: [["count"]],
-            breakout: [
-              ["field", PEOPLE.SOURCE, null],
-              ["field", PEOPLE.CREATED_AT, { "temporal-unit": "year" }],
-            ],
+      visitQuestionAdhoc(
+        {
+          display: "pivot",
+          dataset_query: {
+            type: "query",
+            database: PG_DB_ID,
+            query: {
+              "source-table": PEOPLE_ID,
+              aggregation: [["count"]],
+              breakout: [
+                ["field", PEOPLE.SOURCE, null],
+                ["field", PEOPLE.CREATED_AT, { "temporal-unit": "year" }],
+              ],
+            },
           },
```

Comment:
It *might* clarify things if we set a const for args that are big objects like this.

Then `visitQuestionAdhoc` gets this as an arg, and we nest things less, hoping to make the whole more understandable.

Reply:
Absolutely agreeing with you.
In many places, that's exactly what we've been doing. Especially with the `cy.createQuestion()`. That can be one of the next tasks for the Cypress cleanup.

Thanks for bringing it up, @gusaiani